src/framework/application.js

```diff
@@ -181,7 +182,8 @@ class Application extends AppBase {
             SpriteComponentSystem,
             LayoutGroupComponentSystem,
             LayoutChildComponentSystem,
-            ZoneComponentSystem
+            ZoneComponentSystem,
+            GSplatComponentSystem
```

Comment:
This is one of those lists where it looks random but there are probably some weird reasons for the ordering.

Reply:
yeah it used to be sorted this way and it's possible there is some internal dependency between some of them, so I just preserved it for now.